app/javascript/packs/prompt.ts

```diff
@@ -0,0 +1,25 @@
+import Vue from "vue";
+import Prompt from "../prompt.vue";
+
+document.addEventListener("DOMContentLoaded", () => {
+  const props = {
+    prompt: getPrompt()
+  }
+
+  const comp = new Vue({
+    render: h => h(Prompt, { props })
+  }).$mount();
+  document.body.appendChild(comp.$el);
+});
+
+function getPrompt() {
+  let prompts = document.querySelectorAll('.prompt-external-link');
```

Comment:
We could use `querySelector` instead of `querySelectorAll` here because we only expect to find one of these.